ansible/roles/kubernetes/tasks/debian.yml

```diff
@@ -97,6 +97,11 @@
 # sysctl net.bridge.bridge-nf-call-iptables=1 to pass bridged IPv4 traffic to iptables’ chains.
 # This is a requirement for some CNI plugins to work.
 # https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
+- name: Load br_netfilter
+  modprobe:
```

Comment:
Doing _just_ a `modprobe` does not persist through reboots. As the ansible code is not ran each time, the module load should be made persistent through reboots.

That could be done by a task which writes the appropiate modules file to `/etc/modules-load.d/br_netfilter.conf` (not 100% this is the correct path on Ubuntu). Something like this in short:
```
- copy:
    dest: /etc/modules-load.d/br_netfilter.conf
    content: br_netfilter
  when: modprobe_br_netfilter.changed
```

The example above is untested and does not have to be copied 1 on 1.